### ISSUE TITLE
Lychee trys to parse math equation leading to erroneous error message.

### ISSUE THREAD

**Issue by etpalmer63** [lycheeverse/lychee#1197]
For example:

```
$$
[\psi](\mathbf{L}) 
$$
```

Is interpreted as a link and checked, giving an erroneous error message. 

Is there anyway to workaround this?  -- Thank you!

**Comment by mre** [comment]
I'm assuming that's inside a Markdown file?

> Is there anyway to workaround this? -- Thank you!

Well, you can exclude it with `--exclude mathbf` until we fixed the bug.

**Comment by etpalmer63** [comment]
Ah thank you @mre ! I didn't realize the exclude would be that easy so I hadn't even looked yet. Thanks!

**Comment by HU90m** [comment]
The solution to this bug is to support the maths syntax. We wouldn't really want to [work out a spec](https://github.com/raphlinus/pulldown-cmark/pull/733) and add custom parsers atop `pulldown-cmark`, so we are blocked on this being supported upstream.

It looks like there is some interest: https://github.com/raphlinus/pulldown-cmark/pull/622 https://github.com/rust-lang/mdBook/issues/2222 

As this bug has a workaround, I think it would be ok to close this issue with the stance that we will support maths syntax when `pulldown-cmark` does. Of course, discussion can still take place on the closed issue and it can be re-opened by anyone who wishes.

@mre I will leave closing this to you if you agree.

**Comment by mre** [comment]
Good call. Seems like upstream process on this feature is slow, so might not make much sense to hold our breath before it lands. Closing. In case this gets merged in the future, I'd be thankful if someone could reopen the issue.

**Comment by mre** [comment]
Thanks for the notification.

**Comment by mre** [comment]
👋 It got fixed by Hugo.

**Comment by etpalmer63** [comment]
Thank you both for getting this done! 🙏
